Sources/NIOWebSocket/WebSocketServerUpgrader.swift

```diff
@@ -51,14 +51,14 @@ fileprivate extension HTTPHeaders {
 /// A `HTTPServerProtocolUpgrader` that knows how to do the WebSocket upgrade dance.
 ///
 /// Users may frequently want to offer multiple websocket endpoints on the same port. For this
-/// reason, this `WebSocketUpgrader` only knows how to do the required parts of the upgrade and to
+/// reason, this `WebServerSocketUpgrader` only knows how to do the required parts of the upgrade and to
 /// complete the handshake. Users are expected to provide a callback that examines the HTTP headers
 /// (including the path) and determines whether this is a websocket upgrade request that is acceptable
 /// to them.
 ///
 /// This upgrader assumes that the `HTTPServerUpgradeHandler` will appropriately mutate the pipeline to
 /// remove the HTTP `ChannelHandler`s.
-public final class WebSocketUpgrader: HTTPServerProtocolUpgrader {
+public final class WebSocketServerUpgrader: HTTPServerProtocolUpgrader {
```

Comment:
Changing the name of this class is a semver major change, which means it'll have to wait until SwiftNIO 3.0. That won't come any time soon!

If you want to land this patch sooner, consider adding a deprecated typealias that gives back the old name. That will allow existing code to continue to function. Per our API evolution guidelines, you'll also have to rename this to `NIOWebSocketServerUpgrader`.

Reply:
Okay, will do. Thanks.